pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
     "rosettasciio[hdf5]",
     "scikit-image>=0.18",
     "scipy>=1.6.0",
-    "sympy>=1.6",
+    "sympy>=1.10",
     "tqdm>=4.9.0",
     "traits>=4.5.0",
```

Comment:
This requires a newer traits version that is unreleased, correct?

Reply:
Traits currently doesn't have a release on pypi that supports python 3.13 and this is why when testing for python 3.13, it is necessary to install the development branch, which has the fixes for python 3.13. Hyperspy _doesn't require_ a newer version of traits, but in practise, to install hyperspy on python 3.13 from pypi.org a new version of traits will be necessary. The alternative is to install from the development branch of traits.
Other distribution platform (such as conda-forge, for example), would build traits for python 3.13 using a patch adding python 3.13 support - I didn't check if this is easily feasible but that's a possibility.